bin/buildBlueprints.js

```diff
@@ -69,38 +69,39 @@ function loadBlueprintsFromPath(filePath, isProduction) {
 
 function loadDefaultConfigs(options) {
   console.log('...using default configs');
-  if (options.runTest) {
-    console.log('...Setting up tests:');
-    var config = _.merge(
-      {},
-      configs.DefaultTestingConfig,
-      { webpack: { entry: getWebpackEntryForTest('./') } }
-    );
-    return [ config ];
-
-  } else if (options.client) {
-    console.log('...client');
-    return [ configs.getClientConfig(options.production) ];
-
-  } else if (options.server) {
-    console.log('...server');
-    return [ configs.getServerConfig(options.production) ];
-
-  } else if (options.clientAndServer) {
-    console.log('...both');
-    return [
-      configs.getClientConfig(options.production),
-      configs.getServerConfig(options.production),
-    ];
+  var production = options.env === PRODUCTION_ENV;
```

Comment:
Hey Dave, just creeping on this patch a bit. Do you mind calling this `isProduction`?